pkg/alpine/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # image was bootstraped using FROM lfedge/eve-alpine-base:f11c7a5dcffe1ab7b466bea7e26010a27da62b3b AS cache
```

Comment:
Should we also update this comment?

Reply:
I do not understand completely why eve-alpine-base is needed, but according to the doc:

> The first option requires update of eve-alpine-base image to use another minirootfs and repository and pointing eve-alpine to be based on novel eve-alpine-base using `FROM lfedge/eve-alpine-base`. This action will invalidate all stored packages inside eve-alpine and download them from repository.

Unfortunately the statement from the last sentence ("and download them from repository") is not possible anymore as the go 1.20.1 package has been removed from http://dl-cdn.alpinelinux.org/alpine/edge/community/aarch64/ .

Solutions to that could be, to build go from source or to get the apk from another mirror (perhaps our own).

Reply:
As I did not use eve-alpine-base but based it on eve-alpine, I think the comment can stay the same ...